tests/configs/default.config.ts

```diff
@@ -282,6 +282,26 @@ export default defineConfig({
       target: '../specifications/petstore.yaml',
     },
   },
+  petstoreNamingConventionCamelCase: {
+    input: '../specifications/petstore.yaml',
+    output: '../generated/default/petstore/endpoints.ts',
```

Comment:
Please change the output path for each test case. The same goes for other tests 👀 

```suggestion
    output: '../generated/default/petstore-naming-convention-camel-case/endpoints.ts',
```

Reply:
ok sorry. Done